pandas/tests/arithmetic/test_period.py

```diff
@@ -1525,8 +1525,11 @@ def test_pi_sub_period_nat(self):
 
     @pytest.mark.parametrize("scalars", ["a", False, 1, 1.0, None])
     def test_comparison_operations(self, scalars):
-        # GH 28980
+        # GH #28980 and GH #28981
         expected = Series([False, False])
         s = Series([pd.Period("2019"), pd.Period("2020")], dtype="period[A-DEC]")
         result = s == scalars
         tm.assert_series_equal(result, expected)
+        s = pd.Series([pd.Interval(0, 1), pd.Interval(1, 2)], dtype="interval")
```

Comment:
Can you put this test in /tests/arithmetic/test_interval.py?

Reply:
ok done, a new PR was pushed
[ x ] tests passed
[ x ] black pandas